run-checks

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+mkdir -p bin
+( cd bin &&  wget https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz -O spread.tar.gz && tar xzvf spread.tar.gz )
```

Comment:
how about:
wget https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz -O - | zcat - | tar x

that saves disk space by directly streaming the tarball to disk (and you dont need too clean up temporary files ;) )

Reply:
@ogra1, good idea, thanks